src/cdk-experimental/ui-patterns/behaviors/expansion/expansion.spec.ts

```diff
@@ -242,10 +242,10 @@ describe('Expansion', () => {
       expect(expansion.isExpandable(items[0])).toBeTrue();
     });
 
-    it('should return true if an item is disabled and skipDisabled is false', () => {
+    it('should return false if an item is disabled and skipDisabled is false', () => {
       const {expansion, items} = getExpansion({skipDisabled: signal(false)});
       items[0].disabled.set(true);
-      expect(expansion.isExpandable(items[0])).toBeTrue();
+      expect(expansion.isExpandable(items[0])).toBeFalse();
```

Comment:
I'm ok with this change as long as items in a readonly list are expandable. We should add a todo to add `readonly` to the `ExpansionList` behavior